src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -268,6 +268,17 @@ extern void setDbPrefix(std::string _dbPrefix)
 
 /*****************************************************************************
 *
+* dbPrefixGet -
+*/
+const char* dbPrefixGet(void)
```

Comment:
Let's assume that our functions should return always `char*` (not an opinion yet about that... I have to think about it). Would be the following work?

```
std::string s = getDbPrefix()
```

or I need to write something like:

```
std::string s = std::string(getDbPrefix());
```